elixir/apps/web/assets/js/app.js

```diff
@@ -1,4 +1,20 @@
-import "@fontsource/source-sans-pro"
+import '@fontsource/source-sans-3/200.css';
```

Comment:
Should we use fewer front-weights than importing all of them? I think 2-3 should be enough (regular, bold, and, optionally, light)

Reply:
Good question.  I didn't know what the norm was for most sites and since tailwind allows for all the weights by default I went ahead and imported all of them.  One recommendation I have would be to have at least one more than what you suggested, only because bold was very bold and I have a feeling we probably won't need/want that bold of a font very often (if ever).  Other than that I don't have a strong preference.

Just out of curiosity, is the concern of loading all of them just the amount of time to pull them down to the browser?

Reply:
@bmanifold there are two reasons:
1. Pulling fonts makes UI a bit slower during the first load, which is minor;
2. I don't want us even to have the option of using too many weights as that would be bad for our product consistency. 

But all this is minor, so it's up to you if you want to clear things out for now.

P.S. That "very bold" can be just the weight we define as "our bold", I thin having two bold styles can be a source of mess by itself.